libpod/container_internal.go

```diff
@@ -1057,7 +1057,7 @@ func (c *Container) init(ctx context.Context, retainRetries bool) error {
 
 	// Remove any exec sessions leftover from a potential prior run.
 	if len(c.state.ExecSessions) > 0 {
-		if err := c.runtime.state.RemoveContainerExecSessions(c); err != nil {
+		if err := c.runtime.state.RemoveContainerExecSessions(c); err != nil && !errors.Is(err, define.ErrNoSuchCtr) {
```

Comment:
How are we even in this function if the container has already been removed?

Reply:
Probably got a little too agressive searching for this format.